DuckDuckGo/SettingsViewController.swift

```diff
@@ -134,13 +142,17 @@ class SettingsViewController: UITableViewController {
         versionCell.isUserInteractionEnabled = isDebugBuild
     }
 
-    private func configureDefaultBroswerCell() {
+    private func configureDefaultBrowserCell() {
```

Comment:
Is there actually a difference here? I'm struggling to see it

Reply:
Broswer -> Browser
Typo =D

Reply:
Lol, I would never have spotted that #dyslexic